src/magick-image.ts

```diff
@@ -394,6 +397,18 @@ export class MagickImage extends NativeInstance implements IMagickImage {
         });
     }
 
+    brightnessContrast(brightness: number): void;
+    brightnessContrast(brightness: number, contrast: number): void;
+    brightnessContrast(brightness: number, contrast: number, channels: Channels): void;
+    brightnessContrast(brightness: number, contrastOrUndefined?: number, channels?: Channels): void {
+        const contrast = contrastOrUndefined === undefined? 0 : contrastOrUndefined;
```

Comment:
Could you change this to `undefined ?` for consistency with the rest of the code?